packages/lesswrong/components/hooks/CountItemsContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {
```

Comment:
Each individual subscription type renders a component that loads its own list of subscriptions, which makes it difficult to know at the top-level whether or not the user has _no_ subscriptions. To handle this, we add this context which counts the number of subscriptions that are found in a ref that can be accessed at the top-level.